src/simply/simply_voice.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include "simply_msg.h"
+#include "simply.h"
+
+#include <pebble.h>
+
+#define SIMPLY_VOICE_BUFFER_LENGTH 512
+
+#ifdef PBL_SDK_2
+typedef struct DictationSession DictationSession;
+typedef struct DictationSessionStatus DictationSessionStatus;
+void dictation_session_start(DictationSession *session);
+#endif
```

Comment:
You can move this into util/compat.h, there is a section for `aplite on all versions`. And then you can include "util/compat.h" in simply_voice.c